src/main/java/org/prebid/server/bidder/ix/IxBidder.java

```diff
@@ -409,11 +409,12 @@ private static ExtBidPrebidVideo videoInfo(ExtBidPrebidVideo extBidPrebidVideo)
     private List<FledgeAuctionConfig> extractFledge(IxBidResponse bidResponse) {
         return Optional.ofNullable(bidResponse)
                 .map(IxBidResponse::getExt)
-                .map(IxExtBidResponse::getFledgeAuctionConfigs)
-                .orElse(Collections.emptyMap())
-                .entrySet()
+                .map(IxExtBidResponse::getProtectedAudienceAuctionConfigs)
+                .filter(CollectionUtils::isNotEmpty)
+                .orElse(Collections.emptyList())
                 .stream()
-                .map(e -> FledgeAuctionConfig.builder().impId(e.getKey()).config(e.getValue()).build())
+                .map(ixAuctionConfig -> FledgeAuctionConfig.builder().impId(ixAuctionConfig.getBidId())
+                        .config(ixAuctionConfig.getConfig()).build())
```

Comment:
Please, add `filter(Objects::notNull)` before this map operation.

Reply:
Also, please, refactor a bit:
```
                .map(ixAuctionConfig -> FledgeAuctionConfig.builder()
                         .impId(ixAuctionConfig.getBidId())
                         .config(ixAuctionConfig.getConfig())
                         .build())
```

Reply:
Added nonNull check and reformatted chain calls with line break.